src/openvino_utils.cc

```diff
@@ -330,10 +330,9 @@ ReadParameter(
     std::string* param)
 {
   triton::common::TritonJson::Value value;
-  RETURN_ERROR_IF_FALSE(
-      params.Find(key.c_str(), &value), TRITONSERVER_ERROR_INVALID_ARG,
-      std::string("model configuration is missing the parameter ") + key);
-  RETURN_IF_ERROR(value.MemberAsString("string_value", param));
+  if (params.Find(key.c_str(), &value)) {
+    RETURN_IF_ERROR(value.MemberAsString("string_value", param));
+  }
```

Comment:
I think we should return error, or some way to inform the caller of this function that the parameter is not found, then it is up to the caller to decide whether the parameter is optional.

Reply:
Changed to making the caller log out message when the parameter is not found. We can modify how to handle it If we need to make some of the parameters required in the future.